### PR TITLE
feat: new virtual functions with wrapper methods

### PR DESCRIPTION
**Pull request by KillStr3aK**
This pr adds these methods:

- [x] `CCSPlayer_ItemServices`.`DropActiveWeapon`
- [x] `CCSPlayer_ItemServices`.`RemoveWeapons`
- [x] `CGameSceneNode`.`GetSkeletonInstance`
- [x] `UTIL_CreateEntityByName`
- [x] `CBaseEntity`.`DispatchSpawn`
- [x] `CCSGameRules`.`TerminateRound`

fixes: `CBaseModelEntity_SetModel`

~~Currently working on some more, draft WIP~~ ready.

### CODE REVIEW

**Comment by axe312ger** [comment]
Exactly what we would need. Go for it!

**THANKS FOR INVESTING TIME INTO THIS!!**

File: configs/addons/counterstrikesharp/gamedata/gamedata.json [hunk 1]
```diff
@@ -72,6 +72,13 @@
         "linux": 8
     }
   },
+  "CCSGameRules_TerminateRound": {
+    "signatures": {
+        "library": "server",
+        "windows": "\\x48\\x8B\\xC4\\x4C\\x89\\x48\\x20\\x55\\x56",
```
Comment (KillStr3aK) [inline configs/addons/counterstrikesharp/gamedata/gamedata.json:78]: this signature needs verification as I have no windows setup currently.

File: managed/CounterStrikeSharp.API/Modules/Entities/Constants/RoundEndReason.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with CounterStrikeSharp.  If not, see <https://www.gnu.org/licenses/>. *
+ */
+
+namespace CounterStrikeSharp.API.Modules.Entities.Constants
+{
+    public enum RoundEndReason : uint
```
Comment (KillStr3aK) [inline managed/CounterStrikeSharp.API/Modules/Entities/Constants/RoundEndReason.cs:19]: found in the assembly, using unknown also comes with a devmsg:
`TerminateRound: unknown round end ID %i\n`

File: configs/addons/counterstrikesharp/gamedata/gamedata.json (hunk 1 above)
Comment (KillStr3aK) [reply configs/addons/counterstrikesharp/gamedata/gamedata.json:78]: Yarukon has verified that it is still works.

**Review by roflmuffin — APPROVED** [review]

**Comment by axe312ger** [comment]
Its merged. Awesome. Time for some CS2 modding ❤️

**Comment by miguno** [comment]
Thank you!

**Pull request merged**